subsys/bluetooth/controller/hci/openisa/hci_vendor.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2016 Nordic Semiconductor ASA
```

Comment:
copyright correction

Reply:
Our internal copyright rules say that (C) should never be removed from a file. If a file is significantly changed, then NXP copyright can be added. Since this file is not really changed, I didn't see the need to update the copyright.